Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -130,3 +133,27 @@ extension PostReceiptDataOperation.PostData: Encodable {
     }
 
 }
+
+// MARK: - InitiationSource
+
+extension ProductRequestData.InitiationSource: Encodable, RawRepresentable {
```

Comment:
Hmm I do wonder if this would be better placed in the `ProductRequestData.swift` file... If it's only needed from here, this would be ok though

Reply:
My goal with this is to keep the type definition simpler. This is more of an implementation detail of how the source is posted to the backend.